Formula/wtfutil.rb

```diff
@@ -0,0 +1,72 @@
+class Wtfutil < Formula
+  desc "The personal information dashboard for your terminal"
+  homepage "https://wtfutil.com"
+  url "https://github.com/wtfutil/wtf/archive/v0.10.3.tar.gz"
+  sha256 "8afa15a5729bfa5a8d009e3d03423cf7472e797740b92e6f3e66ca8fd05c8454"
```

Comment:
I think you should probably switch this to git tag & SHA, unless there is a good reason to use a release tarball. It seems that many go projects try to parse git info at build/configure time, so this will have the benefit of allowing that to happen, now or in the future.

Reply:
cool! that is good info for me.